subcommittees/policy-CoC/README.md

```diff
@@ -0,0 +1,33 @@
+# The Code of Conduct Policy Subcommittee
+
+## Purpose and Scope
+
+Enforce the Code of Conduct (CoC) in all parts of the Carpentries community
+
+## What We Do
+
+* communicate with the Carpentries community about the CoC, as well as reporting and enforcement guidelines
+* review and respond to reports of CoC violations as deemed appropriate by the situation
+
+## Contact
+
+* policy@carpentries.org
+* confidential@carpentries.org
```

Comment:
I included language to specify the first is to contact the committee, the second to report